src/Files.Backend/Services/IQuickAccessService.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Files.App.ServicesImplementation
+{
+	public interface IQuickAccessService
+	{
+		/// <summary>
+		/// Gets the list of quick access items
+		/// </summary>
+		/// <returns></returns>
+		Task<List<string>> GetPinnedFoldersAsync();
+
+		/// <summary>
+		/// Pins a folder to the quick access list
+		/// </summary>
+		/// <param name="folderPath"></param>
+		/// <param name="loadExplorerItems"></param>
+		/// <returns></returns>
+		Task PinToSidebar(string folderPath, bool loadExplorerItems = true);
```

Comment:
What does loadExplorerItems do?

Reply:
It was used when I was implementing move to not refresh the item list, but I'll remove it in a sec